tssc/config/decryptors/sops.py

```diff
@@ -0,0 +1,152 @@
+"""ConfigValueDecryptor that uses SOPS to decyrpt ConfigValues
+
+Also See
+--------
+https://github.com/mozilla/sops
+"""
+
+from io import StringIO
+import json
+import os.path
+import re
+import sh
+
+from tssc.config.config_value_decryptor import ConfigValueDecryptor
+
+class SOPS(ConfigValueDecryptor):
+    """ConfigValueDecryptor that uses SOPS to decyrpt ConfigValues
+
+    Parameters
+    ----------
+    additional_sops_args : list
+        Additional arguments to pass to the SOPS command
+
+    Also See
+    --------
+    https://github.com/mozilla/sops
+    """
+
+    SOPS_ENCRYPTED_VALUE_REGEX = r'^ENC\[.*\]$'
```

Comment:
nice tight regex. But, I would explain what it is matching for long term support reasons. specifically calling out that it will anchor on start and end of line.

Reply:
can't everyone just become a regex expert?

Reply:
(i'll add a comment i suppose)